CMakeLists.txt

```diff
@@ -46,3 +46,22 @@ if(BUILD_TESTING)
 	
 	add_subdirectory(tests)
 endif()
+
+# Only for running python tests (with pyTest) using CTest
+# add_subdirectory(ctest-python)
+
+
+message("*** CMAKE_CURRENT_SOURCE_DIR : ${CMAKE_CURRENT_SOURCE_DIR}")
+message("*** python-tests : ${CMAKE_CURRENT_SOURCE_DIR}/ctest-python/python-tests")
+
+enable_testing()
+
+find_package(Python3 EXACT 3.6 COMPONENTS Interpreter)
```

Comment:
No python3.6 is found on my machine. Even worse, python3.6 is not in the official repo.
Suggestions :
- Remove the version constraint
- Use a more recent version, 3.8 for example